src/pymor/reductors/mt.py

```diff
@@ -0,0 +1,156 @@
+# This file is part of the pyMOR project (http://www.pymor.org).
+# Copyright 2013-2020 pyMOR developers and contributors. All rights reserved.
+# License: BSD 2-Clause License (http://opensource.org/licenses/BSD-2-Clause)
+
+from scipy.linalg import eig
+from pymor.algorithms.samdp import samdp
+from pymor.core.base import BasicObject
+from pymor.models.iosys import LTIModel
+from pymor.vectorarrays.numpy import NumpyVectorArray
+from pymor.parameters.base import Mu
+from pymor.reductors.basic import LTIPGReductor
+from pymor.algorithms.gram_schmidt import gram_schmidt, gram_schmidt_biorth
+import numpy as np
+
+
+class MTReductor(BasicObject):
+    """Modal Truncation reductor.
+
+    See Section 9.2 in [A05]_.
+
+    Parameters
+    ----------
+    fom
+        The full-order |LTIModel| to reduce.
+    mu
+        |Parameter values|.
+    """
+
+    def __init__(self, fom, mu=None):
+        assert isinstance(fom, LTIModel)
+        if not isinstance(mu, Mu):
+            mu = fom.parameters.parse(mu)
+        assert fom.parameters.assert_compatible(mu)
+        self.fom = fom
+        self.mu = mu
+        self.V = None
+        self.W = None
+        self._pg_reductor = None
+
+    def reduce(self, r=None, decomposition='samdp', projection='orth',
+               symmetric=False, method_options=None):
+        """Modal Truncation.
+
+        Parameters
+        ----------
+        r
+            Order of the reduced model
```

Comment:
This should be treated as a sentence, and so should end with a dot. Similar below.